packages/testsuite/cypress/support/form-editing.ts

```diff
@@ -60,6 +60,10 @@ Cypress.Commands.add("resetForm", (formId, managementApi, address) => {
   });
 });
 
+Cypress.Commands.add("help", () => {
+  cy.get(":nth-child(2) > a > .form-link-label").click();
```

Comment:
This seems not much descriptive what we are looking for (and maybe also not much effective, starting with all the second children). Is there an element with significant class/id we can start with?

Reply:
Agree :) This is what I found in `npm run develop`. I have changed to `span.form-link-label:contains("Help")`

I believe this is much better and this thread can be resolved.